### PR TITLE
feat(@vtmn/svelte): add `VtmnQuantity` component

### PR DESCRIPTION
**Pull request by Tlahey**
Add a new component `VtmnQuantity` for svelte

image

### CODE REVIEW

**Review by lauthieb — DISMISSED** [review]

File: packages/showcases/svelte/stories/components/selection-controls/VtmnQuantity/VtmnQuantity.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  {parameters}
+  {argTypes}
+/>
+
+<Template let:args>
+  <VtmnQuantity
+    {...args}
+    {error}
+    on:add={() => console.log('Add')}
+    on:subtract={() => console.log('Subtract')}
```
Comment (GaspardMathon) [inline packages/showcases/svelte/stories/components/selection-controls/VtmnQuantity/VtmnQuantity.stories.svelte:23]: You should use the Storybook addon `action`  to display your console messages.

File: packages/showcases/svelte/stories/components/selection-controls/VtmnQuantity/VtmnQuantity.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    on:blur={({ detail: value }) => {
+      if (value < args.min) {
+        error = `Quantity must be upper than ${args.min}`;
```
Comment (GaspardMathon) [inline packages/showcases/svelte/stories/components/selection-controls/VtmnQuantity/VtmnQuantity.stories.svelte:26]:
```suggestion
        error = `Quantity must be greater than ${args.min}`;
```

**Review by GaspardMathon — COMMENTED** [review]
Request changes

**Review by GaspardMathon — APPROVED** [review]

**Review by lauthieb — APPROVED** [review]
Awesome @Tlahey :tada:

**Pull request merged**